interfaces/apparmor/backend.go

```diff
@@ -676,6 +676,9 @@ func addUpdateNSProfile(snapInfo *snap.Info, snippets string, content map[string
 	}
 }
 
+// Allow optional trailing ' ' after "###PROMPT###"
+var promptReplacer = regexp.MustCompile("###PROMPT### ?")
```

Comment:
AFAIU `prompt` modifier only appears at the beginning of the rule, should the regex be anchored at a beginning of the line and should we allow some number of whitespace peceeding it?
```suggestion
var promptReplacer = regexp.MustCompile(`"(?m)^\s+###PROMPT### ?`)
```